apps/extension/webpack.config.js

```diff
@@ -0,0 +1,138 @@
+import dotenv from 'dotenv';
+import path from 'path';
+import CopyPlugin from 'copy-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import webpack from 'webpack';
+
+// Loads default vars from `.env` file in this directory.  If you set
+// environment variables, you will override those defaults.
+dotenv.config();
+
+const definitions = {
+  // process.env.NODE_ENV is automatically provided by DefinePlugin
+
+  PRAX: JSON.stringify(process.env.PRAX),
+  PRAX_ORIGIN: JSON.stringify(`chrome-extension://${process.env.PRAX}`),
+
+  IDB_VERSION: JSON.stringify(Number(process.env.IDB_VERSION)),
```

Comment:
Why would we need to go from string -> number -> string?

Reply:
we could do it directly, but this provides a little bit of type safety, and prevents any creative escape sequence adventures